foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-PurviewPolicySetDefinition.json

```diff
@@ -0,0 +1,213 @@
+        "policies": {
+            "policyDefinitions": [
+                {
+                    "properties": {
+                        "displayName": "Purview should disable public network acess",
+                        "mode": "Indexed",
+                        "description": "Disabling public network access improves security by ensuring that the Purview is not exposed on the public internet. Creating private endpoints can limit exposure of a Function App. Learn more at: https://learn.microsoft.com/en-us/azure/purview/catalog-private-link",
+                        "metadata": {
+                            "version": "1.0.0",
+                            "category": "Purview"
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Audit",
+                                    "Deny",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Audit"
+                            }
+                        },
+                        "policyRule": {
+                            "if": {
+                                "allOf": [
+                                    {
+                                        "field": "type",
+                                        "equals": "Microsoft.Purview/accounts"
+                                    },
+                                    {
+                                        "field": "Microsoft.Purview/accounts/publicNetworkAccess",
+                                        "exists": "true"
+                                    },
+                                    {
+                                        "field": "Microsoft.Purview/accounts/publicNetworkAccess",
+                                        "notEquals": "Enabled"
+                                    }
+                                ]
+                            },
+                            "then": {
+                                "effect": "[[parameters('effect')]"
+                            }
+                        }
+                    },
+                    "name": "Deny-Purview-Public-Network-Access"
+                },
+                {
+                    "properties": {
+                        "displayName": "Private endpoint connections on Azure Purview should be enabled",
+                        "mode": "Indexed",
+                        "description": "Private endpoint connections enforce secure communication by enabling private connectivity to Azure Purview.",
```

Comment:
```suggestion
                        "displayName": "Cross-tenant private endpoint connections for Azure Purview should be restricted",
                        "mode": "Indexed",
                        "description": "Cross-tenant private endpoint connections should be denied for Azure Purview.",
```

Reply:
Auditing of approved private endpoints is not something the platform should worry about in my opinion. They should only worry about cross-tenant private endpoints from a malicious actor. Hence I am proposing this and the updates below.